stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/repository/domain/message/MessageRepository.kt

```diff
@@ -66,6 +66,7 @@ internal class MessageRepositoryImpl(
                 Pagination.LESS_THAN -> {
                     return messageDao.messagesForChannelOlderThan(cid, messageLimit, messageTime)
                 }
+                null -> TODO("Is this just invalid?")
```

Comment:
Looking for some input on this one

Reply:
Can't we do `messageFilterDirection` property "Not Nullable"? :thinking:

Reply:
Looks like we use its `null` value in other places, so that doesn't seem like a possibility.